### PR TITLE
REST API /v2/assets/{id}

### PR DESCRIPTION
**Pull request by algorandskiy**
* New `/v2/assets/{id}` endpoint similar to `/v2/applications/{id}`
* e2e-go test along with existing `/v1/asset/` test

### CODE REVIEW

File: daemon/algod/api/algod.oas2.json [hunk 1]
```diff
@@ -682,6 +682,134 @@
...
+        ],
+        "schemes": [
+          "http"
+        ],
+        "summary": "Get asset information.",
+        "operationId": "GetAssetByID",
+        "parameters": [
+          {
+            "type": "integer",
+            "x-algorand-format": "uint64",
```
Comment (winder) [inline daemon/algod/api/algod.oas2.json:763]: "asset-id" and no need for "uint64", like in the other PR

File: daemon/algod/api/algod.oas2.json [hunk 2]
```diff
@@ -682,6 +682,134 @@
...
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          },
+          "default": {
+            "description": "Unknown Error"
+          }
+        }
+      },
+      "parameters": [
```
Comment (winder) [inline daemon/algod/api/algod.oas2.json:803]: duplicate parameter definition?

File: daemon/algod/api/algod.oas3.yml [hunk 3]
```diff
@@ -798,6 +835,23 @@
         ],
         "type": "object"
       },
+      "AssetInformation": {
+        "description": "Information about the asset including creator",
+        "properties": {
+          "asset": {
+            "$ref": "#/components/schemas/Asset"
+          },
+          "creator": {
```
Comment (winder) [inline daemon/algod/api/algod.oas3.yml:844]: `"x-algorand-format": "Address"`

Comment (algorandskiy) [reply daemon/algod/api/algod.oas3.yml:845]: done

File: daemon/algod/api/algod.oas2.json (hunk 1 above)
Comment (algorandskiy) [reply daemon/algod/api/algod.oas2.json]: done

File: daemon/algod/api/algod.oas2.json (hunk 2 above)
Comment (algorandskiy) [reply daemon/algod/api/algod.oas2.json:800]: all handers has such duplication and I have no idea why and is it really needed or not. I guess the author should address this.

**Pull request merged**